include/wx/textctrl.h

```diff
@@ -742,6 +754,12 @@ class WXDLLIMPEXP_CORE wxTextCtrlBase : public wxControl,
         return GetCompositeControlsDefaultAttributes(variant);
     }
 
+#if wxUSE_SPELLCHECK    
+    // Use native spelling and grammer checking functions.
+    virtual wxTextSpellcheckStatus EnableSpellcheck(const bool enable) = 0;
```

Comment:
No problem with EnableSpellCheck instead of EnableSpellcheck. 

I don't have a strong feeling about EnableSpellChecking either, however when I Google "enable spell checking" it replaces my text with "enable spell check" which (unscientifically) indicates that the former is perhaps more widely used terminology ;)

Reply:
Changed to EnableSpellCheck, removed useless `const` and both `EnableSpellCheck` and `IsSpellCheckEnabled` return false by default.